perception/crosswalk_traffic_light_estimator/include/crosswalk_traffic_light_estimator/node.hpp

```diff
@@ -76,8 +78,12 @@ class CrosswalkTrafficLightEstimatorNode : public rclcpp::Node
   void onTrafficLightArray(const TrafficSignalArray::ConstSharedPtr msg);
 
   void updateLastDetectedSignal(const TrafficLightIdMap & traffic_signals);
+  void updateLastDetectedSignals(const TrafficLightIdMap & traffic_signals);
+  void isFlashing(const TrafficSignal & signal);
+  uint8_t updateState(const TrafficSignal & signal);
   void setCrosswalkTrafficSignal(
-    const lanelet::ConstLanelet & crosswalk, const uint8_t color, TrafficSignalArray & msg) const;
+    const lanelet::ConstLanelet & crosswalk, const uint8_t color, TrafficSignalArray & output,
+    const TrafficSignalArray & msg);
```

Comment:
```suggestion
    const lanelet::ConstLanelet & crosswalk, const uint8_t color,
    const TrafficSignalArray & msg, TrafficSignalArray & output);
```
It's easier to understand to use output in the last